gossip/src/crds_gossip_pull.rs

```diff
@@ -583,6 +568,49 @@ impl CrdsGossipPull {
     }
 }
 
+pub struct CrdsTimeouts<'a> {
+    pubkey: Pubkey,
+    stakes: &'a HashMap<Pubkey, /*lamports:*/ u64>,
+    default_timeout: u64,
+    extended_timeout: u64,
```

Comment:
these are effectively `unstaked_timeout` and `staked_timeout` respectively?

Reply:
We might need to change the timeouts not to be just stake based (e.g. if the node has voted recently or not).
Either way, the commit is just using the existing names.

Reply:
sure. i was just making sure that i deduced their intended usage correctly, since the names, how they're later resolved and eventually used, is inconsistent